extensions/wikia/PotentialMemberPageExperiments/scripts/entry-point-experiment.js

```diff
@@ -74,15 +87,26 @@ require([
 	}
 
 	function experimentEnabled() {
+		experimentGroup = abTest.getGroup(experimentNames[contentLanguage]);
 		return experiments.hasOwnProperty(experimentGroup);
 	}
 
 	function setupExperiment(resources) {
-		var experiment = experiments[experimentGroup];
+		var experiment = experiments[experimentGroup],
+			templateData;
 
 		loader.processStyle(resources.styles);
+		mw.messages.set(resources.messages);
+
+		templateData = {
+			bannerType: experiment.type,
+			language: contentLanguage,
+			messageText: mw.message('pmp-entry-point-message').plain(),
```

Comment:
In general, messages should not contain HTML (both because of security and because translators shouldn't be required to produce valid HTML). In a case like `<strong>` you'd use wikitext `'''` and parse (and `<strong>` is valid wikitext as well), but parsing doesn't work in JS, so you'd have to retrieve the parse messages from the server side. Since it's an experiment though, we can make a temporary exception.